classes/class-ep-debug-bar-elasticpress.php

```diff
@@ -175,68 +179,87 @@ public function render() {
 	}
 
 	/**
-	 * Display debug settings that effect if Elastic Press logs are visible or not.
+	 * Display debug info that effects the Elastic Press logs being visible or not.
 	 */
-	protected function display_debug_settings_info() {
+	protected function display_debugging_info() {
 		?>
 		<div class="qm-boxed">
 			<section>
-				<h3><?php esc_html_e( 'Debug Settings', 'debug-bar' ); ?></h3>
+				<h3><?php esc_html_e( 'Debug Info', 'debug-bar' ); ?></h3>
 				<p>
-					<?php esc_html_e( 'One of the following debug settings should be enabled for search query logs to be displayed', 'debug-bar' ); ?>
+					<?php esc_html_e( 'Elastic Press query logs not being displayed can be due to one of the following reasons:', 'debug-bar' ); ?>
 				</p>
 				<table>
 					<tbody>
-					<tr>
 						<?php
-						// WP_DEBUG - is inactive.
-						$status = false;
-						$message = wp_kses(
-							sprintf( __( '%s is either not defined or turned off', 'debug-bar' ), '<code>WP_DEBUG</code>' ),
-							[ 'code' => [] ]
-						);
-
-						// WP_DEBUG - is active.
-						if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
-							$status  = true;
-							$message = wp_kses(
-								sprintf( __( '%s is defined', 'debug-bar' ), '<code>WP_DEBUG</code>' ),
-								[ 'code' => [] ]
-							);
-						}
-						?>
-						<td>
-							<?php echo $status ? '&#x2714;' : '&#10060;'; ?>
-						</td>
-						<td>
-							<?php echo $message; ?>
-						</td>
-					</tr>
-					<tr>
+						if ( ! defined( WP_DEBUG ) || ! WP_DEBUG || ! defined( WP_EP_DEBUG ) || ! WP_EP_DEBUG ) :
```

Comment:
As discussed I think the logic should be to show this if neither `WP_DEBUG` or `WP_EP_DEBUG` is set, you don't need both.

I think this would need to be `( ! defined( 'WP_DEBUG' ) || ! WP_DEBUG ) && ( ! defined( 'WP_EP_DEBUG' ) || ! WP_EP_DEBUG )`.

Reply:
Additionally `defined()` should have a string passed to it eg. `defined( 'WP_DEBUG' )` - currently this will work but will throw a warning if `WP_DEBUG` or `WP_EP_DEBUG` aren't defined anywhere as you're checking the constants directly.

Reply:
Hm, I thought we agreed to show this message if both are not defined. So that even if one of the constant is defined, the logs will work, so we don't need a message.